lukimgather/settings.py

```diff
@@ -115,11 +148,12 @@
 
 
 # Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/3.2/howto/static-files/
-
-STATIC_URL = '/static/'
+STATIC_LOCATION = "static"
+MEDIA_LOCATION = "media"
+STATIC_URL = "/static/"
+MEDIA_URL = "/media/"
+STATIC_ROOT = os.path.join(BASE_DIR, "static")
+MEDIA_ROOT = os.path.join(BASE_DIR, "media")
 
 # Default primary key field type
-# https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
-
-DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
```

Comment:
It may create bigautofield for dependencies if dependencies do not support 3.2 DEFAULT_AUTO_FIELD till now
```suggestion
DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
```

Reply:
If dependency has missing DEFAULT_AUTO_FIELD it will create migration for external app also